blog/2023-05-18-how-to-use-conda-build.mdx

```diff
@@ -11,106 +11,120 @@ import condabuild from '@site/static/img/blog/2023-05-18-how-to-use-conda-build/
-
+After executing this command, a new directory called `click` will be generated. Inside this directory, you'll find a file named `meta.yaml`. This file contains all the metadata necessary to create a conda package, such as the package name, version number, source code URL, installation instructions, dependencies, license, and more. Note that if you didn't use the `grayskull` command, you'll need to manually write the script yourself. You can make any changes to the `meta.yaml` file and add yourself as the maintainer and distributor of your package.
 
 ## Building the package
+
 To build your `click` package, run this command from the root folder:
+
 ```bash
-conda-build click
+$ conda build click
 ```
 
 After the build process is finished, conda-build will generate a package file in the `conda-bld` directory.
 To locate this file, you can use this command from the terminal:
+
 ```bash 
-conda build click --output
+$ conda build click --output
 ```
+
 You can now install the package locally by running this command in the terminal:
 ```bash 
- conda install --use-local click
- ```
+$ conda install --use-local click
+```
+ 
 Congrats 🎊🎉, you have successfully downloaded your own package! You can modify this package and upload it to <a href="https://anaconda.org/">Anaconda.org</a>.
 
 ### Building a package with C/C++ dependency
+
 With conda-build, it is extremely simple to include non-Python packages. In this example, we will add `gcc` as a build dependency and `mylib` as a runtime dependency. We will also add a new step to compile `mylib.cpp` into a shared library called `mylib.so`.
 
 In the `meta.yaml` file, you can include the following and build the package again:
-```bash
-script: 
-    g++ -o mylib.so -shared -fPIC mylib.cpp
+
+```yaml
+build:
+  script: 
+    - g++ -o mylib.so -shared -fPIC mylib.cpp
+    - cp mylib.so "{{ PREFIX }}/lib"
 
 requirements:
   build:
     - gcc  
- run:
+  run:
     - mylib 
  ```   
 
+:::note
+This example is intentionally simplified teaching purposes. It doesn't address `host` dependencies, build tools like `make`, ABI compatibility, etc.
+For example, conda-build users often rely on Jinja templating to obtain the correct compiler for the platform, instead of hardcoding the compiler (`gcc`) directly.
+Check the [documentation for the `compiler()` function](https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#self-consistent-package-ecosystems).
+For real-world examples, you can check some recipes in [this `conda-forge` search](https://github.com/search?q=org%3Aconda-forge%20compiler(%27c%27)%20path%3Ameta.yaml&type=code).
```

Comment:
```suggestion
This example is intentionally simplified for teaching purposes. It doesn't address `host` dependencies, build tools like `make`, ABI compatibility, etc.
For example, conda-build users often rely on Jinja templating to obtain the correct compiler for the platform, instead of hardcoding the compiler (`gcc`) directly.

Check the [documentation for the `compiler()` function](https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#self-consistent-package-ecosystems).
For real-world examples, you can check some recipes in [this `conda-forge` search](https://github.com/search?q=org%3Aconda-forge%20compiler(%27c%27)%20path%3Ameta.yaml&type=code).
```

Reply:
I'm going off of the assumption that, in spite of the line breaks, this is all one paragraph, so I split it into two for better readability.